src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+import { SystemService } from 'app/system/system.service';
+
```

Comment:
add comment view scheduler job component

Reply:
Please check again. I commented for the first two imports and removed the third one